README.md

```diff
@@ -1,29 +1,37 @@
 # NUMA
 The N User Map Archive
 
-## Development Setup
+
+## First Time Setup
 
 Documentation on working with Python in the Google App Engine can be found here:
 https://cloud.google.com/appengine/docs/python/.    To get running locally,  try
 this:
 
 - Install the [Google Cloud SDK](https://cloud.google.com/sdk/).
-- Check out this repository:  
```

Comment:
Ending a line in two spaces is a Markdown trick to add a line break.  Dunno if you removed those on purpose or if your editor is trying to be helpful and stripping them...

Reply:
Ah, indeed, thanks. Switching to backslashes.